modules/core/src/main/java/org/locationtech/jts/io/WKTOrdinates.java

```diff
@@ -0,0 +1,35 @@
+package org.locationtech.jts.io;
+
+import java.util.EnumSet;
+
+/**
+ * An enumeration of possible Well-Known-Text ordinates
+ */
+public enum WKTOrdinates {
+  /**
+   * X-ordinate
+   */
+  X,
+  /**
+   * Y-ordinate
+   */
+  Y,
+  /**
+   * Z-ordinate
+   */
+  Z,
+  /**
+   * Measure-ordinate
+   */
+  M;
+
+  private static final EnumSet<WKTOrdinates> XY = EnumSet.of(X, Y);
+  private static final EnumSet<WKTOrdinates> XYZ = EnumSet.of(X, Y, Z);
+  private static final EnumSet<WKTOrdinates> XYM = EnumSet.of(X, Y, M);
+  private static final EnumSet<WKTOrdinates> XYZM = EnumSet.of(X, Y, Z, M);
+
+  public static EnumSet<WKTOrdinates> getXY() { return XY.clone(); }
```

Comment:
Do you just want this to be unmodifiable?

Reply:
Yes, is there a better way?

Reply:
Collections.unmodifiableSet( XY ) is what I am used to. Not perfect as it masks that it is an EnumSet, but good enough to prevent modification.

Guava provides an Sets.immutableEnumSet(enumSet) but I do not want the dependency.

Reply:
Recommending calling these createXY, createXYZ, etc... so it is clear a copy is being produced?